prov/efa/src/efa_domain.c

```diff
@@ -278,6 +278,92 @@ void efa_atfork_callback()
 	abort();
 }
 
+/* @brief Check when fork is requested and abort in unsupported cases
+ *
+ * We check if user or another library asked or enabled fork support and
+ * return failure if HUGEPAGES are also enabled as EFA provider does not support this case.
+ *
+ * In addition, we install a fork handler to ensure that we abort if another
+ * library or process initiates a fork and we determined from previous logic 
+ * that we cannot support that.
+ * 
+ * @param domain_fid domain fid so we can check register memory during initialization.
+ * @return error number if we failed to initialize, 0 otherwise
+ */
+#ifndef _WIN32
+
+static
+int efa_initialize_fork_support(struct fid_domain* domain_fid)
```

Comment:
For this win32 version, can you add a similar function header?

Reply:
done